README.md

```diff
@@ -34,5 +34,8 @@ It can be pulled with `docker pull bpfk/stoa`.
 
 ## Manual
 
-One can use `npm start` to run Stoa, and access it on its default port (3836).
+One can use `npm start` to run Stoa, and access it on its default port (4242).
+```sh
+$ npm start -- -c docs/config.example.yaml
+```
```

Comment:
The `config.example.yaml` is not functional. IIRC the previous `npm start` still works ?

Reply:
Just run `npm start` not work.
`130 henry@BOSAGORA ~/work/boa/Github/stoa (git)-[update_doc] % npm start                                         :(

> stoa@0.0.0 prestart
> npm run json2ts


> stoa@0.0.0 json2ts
> ./node_modules/.bin/json2ts -i src/modules/data/schemas/ -o src/modules/data/types/


> stoa@0.0.0 start
> NODE_ENV=production ts-node src/main.ts

EISDIR: illegal operation on a directory, read
npm ERR! code 1
npm ERR! path /Users/henry/Work/BOA/Github/stoa
npm ERR! command failed
npm ERR! command sh -c NODE_ENV=production ts-node src/main.ts

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/henry/.npm/_logs/2020-10-28T09_15_13_032Z-debug.log`

Reply:
Oh I get:
> Config file '/Users/geod24/projects/bpfk/stoa/config.yaml' does not exists

Reply:
Yes,
https://github.com/bpfkorea/stoa/blob/7efdb9af55a91c47bb1b5fe225e3958008b77463/src/modules/common/Config.ts#L83

I will Fix it.